modules/fabric-net-firewall/variables.tf

```diff
@@ -16,53 +16,59 @@
 
 variable "network" {
   description = "Name of the network this set of firewall rules applies to."
+  type        = string
 }
 
 variable "project_id" {
   description = "Project id of the project that holds the network."
+  type        = string
 }
 
 variable "internal_ranges_enabled" {
   description = "Create rules for intra-VPC ranges."
+  type        = bool
   default     = false
 }
 
 variable "internal_ranges" {
   description = "IP CIDR ranges for intra-VPC rules."
+  type        = list(string)
   default     = []
 }
 
 variable "internal_allow" {
   description = "Allow rules for internal ranges."
-  default = [
-    {
-      protocol = "icmp"
-    },
-  ]
+  type        = list(object({ protocol = string, ports = list(string) }))
+  default     = []
```

Comment:
Please don't change the defaults.

Reply:
got it. will resolve this soon. The linters would not pass because I changed the default. But the current default wont allow terragrunt to work. So I am looking for another solution.

Reply:
I had the same issue, and used this as the default. It does change the default but makes terragrunt happy.

```
default = [
    {
      protocol = "icmp"
      ports    = []
    },
  ]
```

I'm not sure what the correct solution is.

Reply:
Does Terragrunt work with `type = list(any)`?